### DISCUSSION TITLE
GraphQL API: projectItems not availavle under issue object

### DISCUSSION THREAD

**Discussion by petrandr** [community/community#73546] [Enterprise]
### Select Topic Area

Bug

### Body

I am running  GitHub Enterprise Server (v3.10.3) on premises and am trying to use the GraphQL in order to automate some workflows.

What i want to do using GraphQL is to fetch the `OPEN` issues under a particular repository along with their assigned projects.

So I created the below query.

```graphql
query GetRepoIssues($owner: String!, $repo: String!, $after: String) {
  repository(owner: $owner, name: $repo) {
    issues(first: 100, after: $after, states: [OPEN]) {
      nodes {
        id
        title
        number
        projectItems(first: 10) {
          nodes {
            project {
              number
              title
            }
          }
        }
      }
      pageInfo {
        endCursor
        hasNextPage
        hasPreviousPage
      }
      totalCount
    }
  }
}
```

When i run it I get the error `Field 'projectItems' doesn't exist on type 'Issue'`, which means there is no such a field under the issue object.

In the documentation of the [Enterprise Server 3.10](https://docs.github.com/en/enterprise-server@3.10/graphql/reference/objects#issue), indeed there is no field `projectItems` but in the documentation of [Free, Pro, & Team](https://docs.github.com/en/graphql/reference/objects#issue) there is.

In the GraphQL change-logs I can see that the field `projectItems` introduce on [2022-06-23](https://docs.github.com/en/graphql/overview/changelog#schema-changes-for-2022-06-23) but up to now it is not available to Enterprise Customers.

Is this a bug? If not then why this field is not available to Enterprise Customers?

**Comment by Naimulnobel** [comment]
The `projectItems` field was indeed introduced to the GitHub GraphQL API on 2022-06-23, as you mentioned. However, updates to the GitHub GraphQL API are not always immediately available in GitHub Enterprise Server. There is often a delay between when a feature is introduced in the cloud version of GitHub and when it becomes available in GitHub Enterprise Server.

This is not a bug, but rather a result of the release process for GitHub Enterprise Server. New features and updates are first introduced in the cloud version of GitHub, and then they are bundled into releases for GitHub Enterprise Server. The timing of these releases depends on a variety of factors, including the complexity of the feature, the stability of the feature in the cloud version of GitHub, and the release schedule for GitHub Enterprise Server.

As of now, the `projectItems` field is not available in GitHub Enterprise Server 3.10.3. It's possible that it will be included in a future release, but I don't have any information on when that might be. For the most accurate and up-to-date information, I would recommend reaching out to GitHub's support or checking the GitHub Enterprise Server release notes regularly.

**Comment by vidursingh** [comment]
It's been almost a year now and this field is still not present in the enterprise version of the API. There is a need for this field, and it seems strange not to include it.
